diff -uraN minitube-3.5/minitube.pro omv-minitube-3.5/minitube.pro
--- minitube-3.5/minitube.pro	2020-08-07 11:22:48.000000000 +0200
+++ omv-minitube-3.5/minitube.pro	2020-08-09 15:06:12.735082669 +0200
@@ -40,7 +40,11 @@
 DEFINES += MEDIA_MPV
 include(lib/media/media.pri)
 
+isEmpty(USE_SYSTEM_QTSINGLEAPPLICATION){
 include(src/qtsingleapplication/qtsingleapplication.pri)
+} else {
+  CONFIG += qtsingleapplication
+}
 include(src/invidious/invidious.pri)
 
 HEADERS += src/video.h \
