diff -uraN minitube-3.4/minitube.pro omv-minitube-3.4/minitube.pro
--- minitube-3.4/minitube.pro	2020-05-30 17:30:47.000000000 +0200
+++ omv-minitube-3.4/minitube.pro	2020-05-31 17:52:35.942077934 +0200
@@ -37,7 +37,11 @@
 DEFINES += MEDIA_MPV
 include(lib/media/media.pri)
 
+isEmpty(USE_SYSTEM_QTSINGLEAPPLICATION){
 include(src/qtsingleapplication/qtsingleapplication.pri)
+} else {
+  CONFIG += qtsingleapplication
+}
 
 HEADERS += src/video.h \
     src/messagebar.h \
