diff -uraN minitube-3.5.1/minitube.pro omv-minitube-3.5.1/minitube.pro
--- minitube-3.5.1/minitube.pro	2020-08-17 16:59:56.000000000 +0200
+++ omv-minitube-3.5.1/minitube.pro	2020-08-19 15:56:40.794028698 +0200
@@ -40,7 +40,11 @@
 DEFINES += MEDIA_MPV
 include(lib/media/media.pri)
 
+isEmpty(USE_SYSTEM_QTSINGLEAPPLICATION){
 include(src/qtsingleapplication/qtsingleapplication.pri)
+} else {
+  CONFIG += qtsingleapplication
+}
 include(src/invidious/invidious.pri)
 
 INCLUDEPATH += $$PWD/src
